HW7/JPHW7.py

```diff
@@ -0,0 +1,55 @@
+## Folium mapping
+
+'''Populating the map with clusters of houses and a heat map of crime rates.'''
+
+#import system modules
+import os
+
+#import data manipulators
+import pandas as pd
+
+#import folium plugins
+from folium.plugins import MarkerCluster
+import folium
+from folium import plugins
+
+#load King County datasets
+crimes = pd.read_csv('King_County_Sheriff_s_Office.csv')
```

Comment:
the data is not in the same folder, you might want to add the path of the data files.